### ISSUE TITLE
I was charged $69.99 and I already purchased my copy years ago

### ISSUE THREAD

**Issue by CatherineHebert** [hymns-for-worship/issues#83]
As is becoming quite the habit, last Sunday during service I was once again not able to access my sheet music (it had been working for maybe a few weeks after not working).  I was presented with an option to purchase one of 3 subscriptions.  I had heard many times people were told Apple would not charge you again if you "buy" it again so I just picked one of them, fully expecting not to be charged.  It wasn't until today that I received an email from Apple with my receipt for payment of $69.99, noting that this was a renewal subscription - that I'd be paying that every year.  I purchased both Hymns for Worship Revised and Hymns for Worship Supplement many years ago - it was not a subscription.  The last 2 years has been off an on (mostly off) of whether I could access the sheet music or not).  I would like the recent "purchase" of $69.99 refunded and I would like my original purchase to work for a change.  If I have to grab a physical songbook every time "just in case" then what is the sense in even having the digital hymns on my iPad.  I might as well just bring my hard copy and use it.

**Comment by rodydavis** [comment]
Refunds go through apple and google.

If you bought the books you can restore them and they will show up. There was a bug fixed that some users had access to the hymnals without a valid purchase.

There have been super really major migrations and slow becoming more stable over time

**Comment by CatherineHebert** [comment]
Apple refunded.  Awesome. Then I was back to the offer of purchasing either free (no sheet music), basic or pro plans.  Didn't go down that road again.  Not sure how I did it but I was able to restore my lifetime (with sheet music) - I think it was in Account settings - there it let me restore my purchases.  It is working for the moment.  My plan is to check before I leave the house every time I'm planning to use it to avoid surprises when services start as we don't have pews and would need to bring a book to my seat if it's not working.
